src/vs/workbench/contrib/terminal/common/terminal.ts

```diff
@@ -207,6 +207,12 @@ export interface ITerminalConfiguration {
 		focusMode: 'singleClick' | 'doubleClick';
 	},
 	bellDuration: number;
+	defaultLocation: TerminalTarget.Editor | TerminalTarget.TerminalView;
+}
+
+export const enum TerminalTarget {
```

Comment:
Can change this to `TerminalLocation` now too. Only reason I went with target is because I wasn't sure if we already had a mention of "location"; we do but it's for tabs location which makes sense.